docs/operate/validators/validator-signatures-aws.mdx

```diff
@@ -10,9 +10,9 @@ If you're only intending to run a Validator for testing or development purposes,
 
 :::
 
-### 1. Create an AWS IAM user and KMS key
+### 1. Create an AWS IAM user, and a signing key
```

Comment:
nit: not a fan of commans in headings but nbd
```suggestion
### 1. Create an AWS IAM user and signing key
```

Reply:
this is on purpose, the signing key isn't AWS-related